child_compassion/models/child_compassion.py

```diff
@@ -300,7 +300,7 @@ class CompassionChild(models.Model):
 
     # Delegation
     ############
-    delegated_to = fields.Many2one('res.partner', 'Delegated to')
+    delegated_to = fields.Many2one('res.users', 'Delegated to')

```

Comment:
This is not good, because it will prevent us to delegate to any sponsor... 
A user is linked to a partner object, so you can fetch it by doing `user.partner_id`.